test/integration/targets/docker_swarm_service/tasks/tests/options.yml

```diff
@@ -1714,67 +1714,6 @@
     - stop_signal_2 is not changed
     - stop_signal_3 is changed
 
-####################################################################
```

Comment:
You might want to also get rid of the superfluous tests which test options which have been moved into option groups, or at least reduce them heavily so that the same tests are not executed twice.

Maybe it's a good idea to remove them from `options.yml` completely, and for each of the suboptions of the option groups in the new test files, add another idempotency test where the old option name is used. I'll add an example below.